src/App.php

```diff
@@ -518,7 +518,7 @@ public function match(Request $request)
      * @param Route $route
      * @return self
      */
-    public function execute(Route $route, array $args = []): self
+    public function execute(Route $route, array $args = [], string $url=''): self
```

Comment:
@lohanidamodar could you please verify how this URL format looks? especially if we have path like: `/category/:id`

Reply:
Ah ok.